### PR TITLE
Add some implementation details for Import and Export

### PR DESCRIPTION
**Pull request by SeanFoongjt**
image
image

### CODE REVIEW

**Comment by SeanFoongjt** [comment]
ExportSequenceDiagram
ImportSequenceDiagram

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -304,6 +304,72 @@ results based on the specified criteria.
 
 <puml src="diagrams/FindSequenceDiagram.puml" alt="FindSequenceDiagram" />
 
+## Export feature
+
+### Implementation
+
```
Comment (lancehaha) [inline docs/DeveloperGuide.md:310]: Shall we add a detail of export command inherit from fileAccessCommand, well not necessary lah

**Comment by charlesliyifeng** [comment]
The export command should not be pushed to the undo stack.

**Review by chashaobao123 — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -94,11 +94,7 @@ The sequence diagram below illustrates the interactions within the `Logic` compo
 
 <puml src="diagrams/DeleteSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `delete 1` Command" />
 
-<box type="info" seamless>
-
-**Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline continues till the end of diagram.
-
-</box>
+> **Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline continues till the end of diagram.
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:97]: paiseh but pls revert to box!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -219,9 +215,11 @@ Step 2. The `edit` command updates the details of the contact with index 1 to ha
 The following sequence diagram shows how an `edit` command goes through the `Logic` component:
 
 <puml src="diagrams/EditSequenceDiagram.puml" alt="EditSequenceDiagram" />
-<box type="info" seamless>
 
-**Note:** There are no destroy markers (X) for `EditCommand` as it is preserved in the `undo` command stack.
+<br>
+
+> **Note:** There are no destroy markers (X) for `EditCommand` as it is preserved in the `undo` command stack.
+
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:222]: here too :"D

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -273,11 +271,7 @@ The following sequence diagram shows how a `clean` command goes through the `Log
 
 <puml src="diagrams/CleanSequenceDiagram.puml" alt="CleanSequenceDiagram-Logic" />
 
-<box type="info" seamless>
-
-**Note:** There are no destroy markers (X) for `CleanCommand` and `GradYearPredicate` as they are preserved in the `undo` command stack.
-
-</box>
+>> **Note:** There are no destroy markers (X) for `CleanCommand` and `GradYearPredicate` as they are preserved in the `undo` command stack.
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:274]: here too :"D

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -354,15 +359,26 @@ The following sequence diagram shows how a `export` command goes through the `Lo
...
+The `import` command extends `FileAccessCommand` and by extension, `Command`. It also implements `Undoable`. `import` loads data from a save file into DorManagerPro, with the file path of the save file provided by the user. The `import` command is undoable.
 
 Given below is an example usage scenario and how the `import` command behaves at each step.
 
 Step 1. The user executes `import fp/./data/SaveFile3.json`
 
-Step 2. The
+Step 2. The `import` command locates the save file via the file path and reads the data in the save file into DorManagerPro if it is of the correct format and has valid data.
+
+>> **Note:** An error message is raised if the file path does not exist in the device or if the file itself cannot be read into DorManagerPro.
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:370]: here too :"((

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -354,15 +359,26 @@ The following sequence diagram shows how a `export` command goes through the `Lo
...
+
+The following sequence diagram shows how an `import fp/./data/SaveFile3.json` command goes through the `Logic` component:
+<puml src="diagrams/ImportSequenceDiagram.puml"
+alt="ImportSequenceDiagram" />
+
+> **Note:** There is no destroy marker (X) for `ImportCommand` as it is preserved in the `undo` command stack. <br>
+> The lifeline for `ImportCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:378]: here too :"(((((((( (my bad)

**Review by chashaobao123 — APPROVED** [review]

**Pull request merged**